projects/ion/src/lib/button/button.component.ts

```diff
@@ -75,9 +75,11 @@ export class ButtonComponent implements OnInit {
 
   handleSelect(selectedItems: DropdownItem[]): void {
     this.selected.emit(selectedItems);
+    this.showDropdown = false;
 
     if (this.multiple) {
       this.updateBadgeValue(selectedItems);
+      this.showDropdown = true;
```

Comment:
Try it. This way you not need change 2 times the var

```suggestion    
    if (this.multiple) {
      this.updateBadgeValue(selectedItems);
      return;
    }
    
    this.showDropdown = false;
```